tests/fixtures/rendering-tests/template-import-test.gjs

```diff
@@ -1,7 +1,8 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
-import TemplateImport from "my-addon/components/template-import";
+
+import TemplateImport from "#src/components/template-import";
```

Comment:
if we're only using #src for our own test fixtures let's just use a known addon name like we've been doing up until now

Reply:
you made me check, but this _was erroring_, and is no longer.

I'll will move package.json#imports to another PR

Reply:
https://github.com/ember-cli/ember-addon-blueprint/pull/47